crates/matrix-sdk/src/room/mod.rs

```diff
@@ -952,6 +952,43 @@ impl Room {
         Ok(true)
     }
 
+    /// Set the given account data event for this room.
```

Comment:
That's a bit terse, could you expand this with an example?

Likewise for the other method.